src/main/java/com/aws/greengrass/clientdevices/auth/configuration/SecurityConfiguration.java

```diff
@@ -26,8 +26,9 @@ public final class SecurityConfiguration {
     private static final Logger logger = LogManager.getLogger(SecurityConfiguration.class);
     public static final String SECURITY_TOPIC = "security";
     public static final String CLIENT_DEVICE_TRUST_DURATION_MINUTES_TOPIC = "clientDeviceTrustDurationMinutes";
-    // TODO: change default value to zero (opt-in offline auth)
-    public static final int DEFAULT_CLIENT_DEVICE_TRUST_DURATION_MINUTES = 60;
+    // opt-in trust verification: metadata trust verification is enabled when configured trust duration is non-zero
+    // trust verification is disabled by default
+    public static final int DEFAULT_CLIENT_DEVICE_TRUST_DURATION_MINUTES = 0;
```

Comment:
Should we add one test that ensures that when not provided this defaults to 0?

Reply:
Also, checking this again, we are now allowing people to set the value to 0.1 which might cause undesired behaviour when offline

Reply:
Test for fallback to default when not provided is here: [link](https://github.com/aws-greengrass/aws-greengrass-client-device-auth/blob/main/src/test/java/com/aws/greengrass/clientdevices/auth/configuration/SecurityConfigurationTest.java#L45-L49)

We currently allow only integer values for trust duration config. Decimal numbers are floor-rounded to (cast to) Integers. I just added a unit test for that.